mappings/net/minecraft/entity/player/PlayerEntity.mapping

```diff
@@ -64,9 +77,14 @@ CLASS net/minecraft/class_1657 net/minecraft/entity/player/PlayerEntity
 	METHOD method_7258 isSpawnForced ()Z
 	METHOD method_7259 incrementStat (Lnet/minecraft/class_3445;)V
 		ARG 1 stat
+	METHOD method_7260 increaseRidingMotionStats (DDD)V
+		ARG 1 x
+		ARG 3 y
+		ARG 5 z
```

Comment:
```suggestion
		ARG 1 vx
		ARG 3 vy
		ARG 5 vz
```

Reply:
velocityX etc?

Reply:
I feel like vx is clear enough, but velocityX works too.